clients/apps/web/src/components/Checkout/CheckoutForm.tsx

```diff
@@ -342,29 +345,48 @@ const BaseCheckoutForm = ({
                     )}
                   </FormItem>
 
-                  <FormField
-                    control={control}
-                    name="customer_tax_id"
-                    render={({ field }) => (
-                      <FormItem>
-                        <div className="flex flex-row items-center justify-between">
-                          <FormLabel>Tax ID</FormLabel>
-                          <span className="dark:text-polar-500 text-xs text-gray-500">
-                            Optional
-                          </span>
-                        </div>
-                        <FormControl>
-                          <Input
-                            type="text"
-                            autoComplete="off"
-                            {...field}
-                            value={field.value || ''}
-                          />
-                        </FormControl>
-                        <FormMessage />
-                      </FormItem>
-                    )}
-                  />
+                  {!showTaxId && (
+                    <button
+                      onClick={() => {setShowTaxID(true)}}
+                      className='w-full flex justify-end'
+                    >
+                      <span className='text-xs text-blue font-medium' >Add Tax ID</span>
+                    </button>
+                  )}
+
+                  {showTaxId && (
+                    <FormField
+                      control={control}
+                      name="customer_tax_id"
+                      render={({ field }) => (
+                        <FormItem>
+                          <div className="flex flex-row items-center justify-between">
+                            <FormLabel>Tax ID</FormLabel>
+                            <span className="dark:text-polar-500 text-xs text-gray-500">
+                              Optional
+                            </span>
+                          </div>
+                          <FormControl>
+                            <div className='relative flex'>
+                              <Input
+                                type="text"
+                                autoComplete="off"
+                                {...field}
+                                value={field.value || ''}
+                              />
+                              <button
+                                onClick={() => {setShowTaxID(false)}}
+                                className="absolute right-2 top-2 text-gray-400 hover:text-gray-200"
+                              >
+                                <X size={16} />
```

Comment:
That's the idea, but we already have an icon library. Please remove `lucide-react` and use that instead:

```tsx
import { CloseOutlined } from '@mui/icons-material'

<CloseOutlined className="h-4 w-4" />
```

Reply:
Thanks for the suggestion! I've made the changes.